css/style.css

```diff
@@ -0,0 +1,1381 @@
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+  vertical-align: baseline;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+  cursor: pointer;
+}
+
+.hidden {
+  display: none;
+}
+
+.black-navbar,
+.white-navbar {
+  display: none;
+}
+
+.ind-header {
+  padding: auto 1em;
+  display: flex;
+  justify-content: space-between;
+}
+
+header {
+  display: block;
+}
+
+.navbar-mobile {
+  display: flex;
+  justify-content: space-between;
+  height: 48px;
+  position: relative;
+  background-color: white;
+}
+
+.navbar-mobile a {
+  width: 106px;
+  height: 100%;
+  background: linear-gradient(to bottom right, #c09, #f42 85%);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.navbar-mobile img {
+  display: block;
+  height: 16px;
+  position: relative;
+  width: 66px;
+}
+
+.navbar-mobile ul {
+  width: 96px;
+  display: flex;
+  margin-right: 9px;
+}
+
+.navbar-mobile li {
+  width: 50%;
+  border-left: 1px solid rgba(80, 80, 80, 0.1);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.spacer {
+  height: 60px;
+  font-size: 14px;
+  background-color: rgb(255, 205, 41);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 48px;
+}
+
+.fixed {
+  position: fixed;
+  width: 100%;
+  top: 0;
+}
+
+.insights {
+  display: flex;
+  flex-direction: column;
+  padding: 8px;
+  margin-top: 106px;
+}
+
+.insights li {
+  background-size: cover;
+  background-position: 50%;
+  width: 100%;
+  margin-bottom: 10px;
+  padding: 10px;
+  min-height: 30vh;
+}
+
+.insights li img {
+  display: none;
+}
```

Comment:
This selector has the same style definition of selectors on lines 113 and 198. Group selectors, or create a new class.

Reply:
I am sorry, but this is not possible because the linters will provide that 'Expected selector ".navbar-mobile img" to come before selector ".insights li img" '